_rules/bd5de3.md

```diff
@@ -0,0 +1,37 @@
+
+---
+name: Focusable element has visible focus
+test_type: atomic
+
+ description: |
+  This rule checks that all elements that can receive focus has a focus styling that is visibly different from the normal styling.
+ success_criterion: 
+- 2.4.7 # Focus Visible
+
+ test_aspects:
+- DOM Tree
+- CSS Styling
+
+ authors:
+- Anne Thyme Nørregaard
+---
+
+## Applicability
+
+This rule applies to all elements that can receive focus.
+
+## Expectation
+
+When the target element's focus state is triggered, a [visible](#visible) change in text style, border, box-shadow, background-image or color occurs for the element.
```

Comment:
I think it is better to allow any style change to indicate focus, not just these specific examples. For instance a menu could show the focused item by making the text larger or indenting the menu item that has received focus, making it stand out from it's vertically aligned siblings.